src/main/java/org/opensearch/performanceanalyzer/commons/metrics/AllMetrics.java

```diff
@@ -943,7 +943,9 @@ public enum SearchBackPressureStatsValue implements MetricValue {
                 SearchBackPressureStatsValue.Constants.SEARCHBP_SEARCH_BACK_PRESSURE_STATS_MODE),
         SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS(
                 SearchBackPressureStatsValue.Constants
-                        .SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS);
+                        .SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS),
+        SEARCHBP_TABLE_NAME(SearchBackPressureStatsValue.Constants.SEARCHBP_TABLE_NAME),
```

Comment:
Why are we replicating this information ?

Reply:
`.SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS); `change to 
`.SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS),` because it is not the final constants in the list
`SEARCHBP_TABLE_NAME(SearchBackPressureStatsValue.Constants.SEARCHBP_TABLE_NAME),` use this for easier access and consistent coding style.